physics/relativity_equation/java/RelativityEquation.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Created by robertoduessmann on 03/10/17.
+ */
+public class RelativityEquation {
+
+	private static final double SPEED_LIGHT = 299792458;
+
+	public static double getEnergy(double mass) {
+		double energy = mass * Math.pow( SPEED_LIGHT, 2 );
```

Comment:
Do you need a local variable when we just return immediately?

Reply:
Yeah, let's do that. It's clear this is energy. Maybe have a comment saying e = m*c^2

Reply:
Great, don't need! 
I created only to try be clearer, but isn't necessary... I will remove :+1: